ihatemoney/models.py

```diff
@@ -246,13 +246,29 @@ def get_bills_unordered(self):
 
     def get_bills(self):
         """Return the list of bills related to this project"""
+        return self.ordered_bills(self.get_bills_unordered())
+
+    def ordered_bills(self, query):
         return (
-            self.get_bills_unordered()
-            .order_by(Bill.date.desc())
+            query.order_by(Bill.date.desc())
             .order_by(Bill.creation_date.desc())
             .order_by(Bill.id.desc())
         )
 
+    def get_bill_weights(self):
```

Comment:
Add a comment about what this function returns, it's not exactly clear (it returns a list of tuples?)